src/domain/individualCustomer/boatsCard/boatsCardUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { BoatCertificateType } from '../../../@types/__generated__/globalTypes';
+import { Boat, BoatCert, LargeBoat } from '../types';
+
+const LARGE_BOAT_ID = '8';
+
+export const isLargeBoat = (boat: LargeBoat | Boat): boat is LargeBoat =>
+  (boat as LargeBoat).boatType.id === LARGE_BOAT_ID;
+
+export const getCertificate = (
+  certificates: BoatCert[],
+  certificateType: BoatCertificateType
+): BoatCert => {
+  return certificates.filter(
+    cert => cert.certificateType === certificateType
+  )[0];
+};
+
+export const getFilename = (url: string | null): string | undefined => {
+  if (!url) {
+    return undefined;
+  }
+
+  return url.split('/').pop();
+};
```

Comment:
I wonder if we can just use [Node's `path` module](https://nodejs.org/api/path.html#path_path_basename_path_ext).

`path.basename('https://venepaikka-api.test.hel.ninja/dummy.pdf')`

Reply:
We would need to install this dependency:
https://github.com/browserify/path-browserify
Maybe not worth for this small thing? :)